<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExecutionLogWidget</class>
 <widget class="QWidget" name="ExecutionLogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabConsole">
     <property name="tabPosition">
      <enum>QTabWidget::TabPosition::East</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="tabGeneralLayout"/>
     </widget>
     <widget class="QWidget" name="tabBan">
      <attribute name="title">
       <string>Blocked IPs</string>
      </attribute>
      <layout class="QVBoxLayout" name="tabBanLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
